cores/arduino/stm32/usb/usbd_conf.c

```diff
@@ -55,9 +55,7 @@ void HAL_PCD_MspInit(PCD_HandleTypeDef *hpcd)
   /* Enable VDDUSB on Pwrctrl CR2 register*/
 #if !defined(STM32WBxx)
   if (__HAL_RCC_PWR_IS_CLK_DISABLED()) {
-    __HAL_RCC_PWR_CLK_ENABLE();
     HAL_PWREx_EnableVddUSB();
-    __HAL_RCC_PWR_CLK_DISABLE();
   } else
 #endif
   {
```

Comment:
As the PWR clock is considered enabled no need to test

Reply:
Right. Fixed.